lib/controllers/coinstac/consortia.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const config = require('config'); // jshint ignore:line
+const _ = require('lodash');
+const consortiaDbOptions = config.get('coinstac.pouchdb.consortia');
+
+const internals = {};
+
+internals.consortiaSchema = joi.object().keys({
+    label: joi.string().required(),
+    users: joi.array().required(),
+    description: joi.string().required(),
+    tags: joi.array().required(),
+    analyses: joi.array().required()
+});
+
+internals.savedConsortiaSchema = internals.consortiaSchema.keys({
```

Comment:
Yes, it does. I wasn't sure, but tested it and it it does append the keys to the existing keys.

https://github.com/hapijs/joi#objectkeysschema